object_info[] bounding_box